modules/nf-core/muscle5/super5/tests/main.nf.test

```diff
@@ -0,0 +1,37 @@
+// nf-core modules test muscle5/super5
+nextflow_process {
+
+    name "Test Process MUSCLE5_SUPER5"
+    script "../main.nf"
+    process "MUSCLE5_SUPER5"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "muscle5"
+    tag "muscle5/super5"
+
+    test("sarscov2 - fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [[ id:'test' ],
+                            fasta = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
+                           ]
+                """
+            }
+        }
+
+         then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.alignment[0][1][0]).match("aln0")},
+                { assert snapshot(process.out.alignment[0][1][1]).match("aln1")},
+                { assert snapshot(process.out.alignment[0][1][2]).match("aln2")},
+                { assert snapshot(process.out.alignment[0][1][3]).match("aln3")},
```

Comment:
@sateeshperi is there another way of doing this? In pytest it was implemented like [this](https://github.com/nf-core/modules/blob/e5472a8d7a0399f1dc3e479433f67cf5e20fc3ff/tests/modules/nf-core/muscle5/super5/test.yml#L7-L14)

Reply:
Sounds about right...I would have done the same...lgtm!